lib/countly.js

```diff
@@ -1342,12 +1340,17 @@ Countly.Bulk = Bulk;
             }
             var req = protocol.request(options, function(res) {
                 var str = "";
+                // to check if result field exists in response
+                var result = false;
                 res.on("data", function(chunk) {
                     str += chunk;
+                    if (chunk.toString().indexOf("result")) {
```

Comment:
just move this to the "isResponseValid" call so that all "request checking" logic is in one place